pyseir/inference/fit_results.py

```diff
@@ -0,0 +1,25 @@
+import os
+from pyseir.load_data import load_county_metadata
+from pyseir import OUTPUT_DIR
+import pandas as pd
+from datetime import datetime
+
+
+def load_t0(fips):
+    """
```

Comment:
This is probably worth a discussion elsewhere, but I'd advocate for docstrings in the style from the [google style guide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods). 

I know that docstring formats can be very controversial :D and I don't care too much right now (the fact that all the functions are well documented is amazing), but would be good to get a consistent format for others reading the code.

Also, since we're using python 3.7, we're trying to do more type hints.  That way you don't have to include them in the docstring itself. [here's](https://google.github.io/styleguide/pyguide.html#319-type-annotations) a good summary!

Reply:
Without a converter I am probably unwilling to spend hours rewriting these :).  I believe the other 3 DS people probably also prefer numpy style so we will have to have a battle. Most docstring formats are interchangeable in documentation generation as well (though obviously nice to have for code cleanliness).

For type hints, I can try to start that though I am bound to forget.  IDEs like pycharm actually read type hints from docstrings so we have resisted so far at my place of work.